core/ordering/cosipbft/pbft/mod.go

```diff
@@ -53,6 +55,29 @@ func (s State) String() string {
 	}
 }
 
+// defines prometheus metrics
+var (
+	promBlocks = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "dela_cosipbft_blocks_total",
+		Help: "total number of blocks",
+	})
+
+	promTxs = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "dela_cosipbft_transactions_block",
+		Help: "total number of transactions in the last block",
+	})
+
+	promRejectedTxs = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "dela_cosipbft_transactions_rejected_block",
+		Help: "total number of rejected transactions in the last block",
+	})
```

Comment:
should we only look at the last block (and possibly use a histogram), or the total number of reject transactions (of all past blocks) and use a counter for this ?